packages/select/src/Select.js

```diff
@@ -90,6 +92,15 @@ const Select = ({
     _cacheUniq = [_cacheUniq];
   }
 
+  attributes.placeholder = (
```

Comment:
I'm not a fan of adding this to the `attributes` object. Instead, can you do `const placeholder = ...` and then pass that to the `react-select` component?

Reply:
yea, I was on the fence about it. Was trying to avoid an extra prop if I could but I guess it's unavoidable.

Unrelated, but do you know what's going on with the PR builds? I tried twice but it said it timed out.

Reply:
I want to move away from using the `attributes` spreading pattern or otherwise I think your solution is a good.

I'm not sure what's up with the builds. A few others have failed too. I'll look into it